packages/admin-ui/ui/src/domain/gift-cards/details/index.tsx

```diff
@@ -76,94 +70,98 @@ const GiftCardDetails = () => {
+              }
+              actionables={actions}
+            >
+              <div className="flex justify-between">
+                <div className="flex space-x-6 divide-x">
+                  <div className="flex flex-col">
+                    <div className="inter-smaller-regular text-grey-50 mb-1">
+                      Original amount
+                    </div>
+                    <div>
+                      {formatAmountWithSymbol({
+                        amount: giftCard.value,
+                        currency: giftCard.region.currency_code,
+                      })}
+                    </div>
                   </div>
-                  <div>
-                    {formatAmountWithSymbol({
-                      amount: giftCard?.value,
-                      currency: giftCard?.region.currency_code,
-                    })}
+                  <div className="flex flex-col pl-6">
+                    <div className="inter-smaller-regular text-grey-50 mb-1">
+                      Balance
+                    </div>
+                    <div>
+                      {formatAmountWithSymbol({
+                        amount: giftCard.balance,
+                        currency: giftCard.region.currency_code,
+                      })}
+                    </div>
                   </div>
-                </div>
-                <div className="flex flex-col pl-6">
-                  <div className="inter-smaller-regular text-grey-50 mb-1">
-                    Balance
-                  </div>
-                  <div>
-                    {formatAmountWithSymbol({
-                      amount: giftCard?.balance,
-                      currency: giftCard?.region.currency_code,
-                    })}
+                  {giftCard.ends_at && (
+                    <div className="flex flex-col pl-6">
+                      <div className="inter-smaller-regular text-grey-50 mb-1">
+                        Expires on
+                      </div>
+                      <div>
+                        {moment(giftCard.ends_at).format("DD MMM YYYY")}
+                      </div>
+                    </div>
+                  )}
+                  <div className="flex flex-col pl-6">
+                    <div className="inter-smaller-regular text-grey-50 mb-1">
+                      Created
+                    </div>
+                    <div>
+                      {moment(giftCard.created_at).format("DD MMM YYYY")}
+                    </div>
                   </div>
                 </div>
-                <div className="flex flex-col pl-6">
-                  <div className="inter-smaller-regular text-grey-50 mb-1">
-                    Created
-                  </div>
-                  <div>
-                    {moment(giftCard?.created_at).format("DD MMM YYYY")}
-                  </div>
+                <div className="flex items-end">
+                  <Badge variant="default">{giftCard?.region?.name}</Badge>
```

Comment:
thought: Should we move this to follow the other details format and add Region as title? Seems to be floating for its self here

Reply:
I like it will move it over to the rest